x-pack/plugins/apm/scripts/bootstrap_apm/src/run.sh

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env bash
+set -euo pipefail
+scriptdir="$(dirname "$0")"
+cd "$scriptdir"
+source variables.sh
+source utils/logger.sh
+
+helpFunction()
+{
+   echo ""
+   echo "Usage: $0 -v "7.17.0" -n my-deployment -s 01_simple_trace.ts -o '--from=now-24h --to=now' "
+   echo -e "\t-v Version of the stack "
+   echo -e "\t-n Deployment name"
+   echo -e "\t-r Region"
+   echo -e "\t-p Hardware profile"
+   echo -e "\t-s Data scenario for apm-synthtrace. Available scenarios can be found https://github.com/elastic/kibana/tree/main/packages/elastic-apm-synthtrace/src/scripts/examples"
+   echo -e "\t-o Options for apm-synthtrace. Supported options can be found https://github.com/elastic/kibana/tree/main/packages/elastic-apm-synthtrace#cli"
+   exit 0 # Exit script after printing help
+}
+
+if ! [ -x "$(command -v ecctl)" ]; then
+  show_msg "[ERROR] ecctl not found. Download it here: https://www.elastic.co/downloads/ecctl " 4
+  exit 1
+fi
+
+if ! test -f "$ECCTL_CONFIG"; then
+   show_msg "[ERROR] Configuration file $ECCTL_CONFIG is missing." 4
+   exit 1
+fi
+
+while getopts ":v:n:r:p:s:o:" opt
+do
+   case "$opt" in
+      v ) VERSION="$OPTARG" ;;
+      n ) DEPLOYMENT_NAME="$OPTARG" ;;
+      r ) REGION="$OPTARG" ;;
+      p ) HARDWARE_PROFILE="$OPTARG" ;;
+      s ) SCENARIO_FILE="$OPTARG" ;;
+      o ) SYNTHTRACE_OPTIONS="$OPTARG" ;;
+      ? ) helpFunction ;; 
+   esac
+done
+
+TMPFILE=$(mktemp /tmp/output-XXXX)
```

Comment:
Nitpick but this doesn't hardcodes `/tmp` 
```suggestion
TMPFILE=$(mktemp -t output-XXXX)
```

Reply:
updated https://github.com/elastic/kibana/pull/123791/commits/def7d17beff884d1903926a5b1bc7b51d990c442